crates/turbopack-dev/src/chunking_context.rs

```diff
@@ -180,22 +180,26 @@ impl ChunkingContext for DevChunkingContext {
 
     #[turbo_tasks::function]
     async fn chunk_path(&self, ident: AssetIdentVc, extension: &str) -> Result<FileSystemPathVc> {
-        fn clean(s: &str) -> String {
+        fn clean_separators(s: &str) -> String {
             s.replace('/', "_")
         }
+        fn clean_additional_extensions(s: &str) -> String {
+            s.replace('.', "_")
+        }
```

Comment:
`next start` won't serve files that start with a `.`, hence this particular change (which is duplicated on the `turbopack-build` side).

TODO: write a comment to explain this.